sharktank/sharktank/ops/sharded_impls.py

```diff
@@ -1633,7 +1633,7 @@ def unflatten_split(
 
 @unshard.override(ReplicatedTensor)
 def unshard_replicated(input: ReplicatedTensor) -> Tensor:
-    return input.shards[0]
+    return input.shards[0].as_torch()
```

Comment:
@sogartar

Reply:
Context: All of the other `unshard()` implementations are returning a torch.Tensor, this one returns a PrimitiveTensor. Do we want to be returning torch.Tensor or PrimitiveTensor from all of them?